src/eddymotion/model.py

```diff
@@ -1,21 +1,19 @@
 """A factory class that adapts DIPY's dMRI models."""
 import warnings
-from copy import deepcopy
-from concurrent.futures import ThreadPoolExecutor, as_completed
-from os import cpu_count
+from joblib import Parallel, delayed
 
 import numpy as np
 from dipy.core.gradients import check_multi_b, gradient_table
 
 
-def _exec_fit(model, data):
+def _exec_fit(model, data, block=None):
```

Comment:
I would prefer "chunk" to "block" because the latter sounds too much like "blocking execution", which is semantically related but not what is intended here.